api/app/signals/apps/api/v1/views/pdf.py

```diff
@@ -61,6 +66,8 @@ class GeneratePdfView(SingleObjectMixin, PDFTemplateView):
     template_name = 'api/pdf/print_signal.html'
     extra_context = {'now': timezone.datetime.now(), }
 
+    max_size = 800
```

Comment:
Maybe add to the settings? With a default of 800.

Reply:
I guess we could.

Reply:
I added a setting to `base.py`